client/question/services/SolutionHandlerServiceSpec.js

```diff
@@ -355,6 +355,29 @@ describe('SolutionHandlerService', function() {
           done();
         });
       });
+
+      it('should return 1st errorLineNumber, when multiple errors', function() {
```

Comment:
Add done as an arg here and call it on line 379 when all async processing finishes. See the [Jasmine docs](https://jasmine.github.io/2.0/introduction#section-Asynchronous_Support) and the other existing tests in this file.

Reply:
done().